### DISCUSSION TITLE
back to full screen

### DISCUSSION THREAD

**Discussion by Quimap** [AppGround-io/bluetooth-keyboard-and-mouse-support#171] [Bugs & Problems]
Currently if I start the app in full screen mode and swipe up to down, the full screen exits and I have to restart the app to get the full screen back.

Could you please add a feature (e.g. button, time interval or swipe gesture) to go back to full screen?

**Comment by zorazorr** [comment]
As workaround you can rotate your phone portrait/landscape. After that it should have full screen again.

But an extra button or timer sounds nice :+1:

**Reply by Quimap** [reply]
thx
